CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog Gitarro
 
+## 0.1.86
+
+- triggered_by_pr_number method removed, we will use --PR parameter only to don't iterate through all open PRs
```

Comment:
For the removal imho we can use a line. Split this in 2 line since are 2 Concepts. Removal and addition